Marlin/pins_MIGHTYBOARD_REVE.h

```diff
@@ -220,3 +231,19 @@
 #define SDSS                53 // B0
 #define SD_DETECT_PIN       30 // C7
 
+
+
+// Rev B additions
+
+//
+//  M3, M4 & M5 support
+//
+#if ENABLED(SPINDLE_ENABLE)
+  #define SPINDLE_ENABLE_PIN 66 // K4   should be pulled up
+  #define SPINDLE_SPEED_PIN   8 // H5   needs a hardware PWM
+  #define SPINDLE_DIR_PIN    67 // K5
+#endif
```

Comment:
As above, define these pins unconditionally.

Mightyboard is still not working, and might never, but we'll see! Thank you for the B mosfet correction.